tests/test_sails.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+import requests
+import json
+
+"""
+TEST RESPONSES
+Includes all status code responses for both get and post requests.
+"""
+
+def test_get_no_param_200_code():
+  """
+  SUMMARY:  Test that GET method returns a 200 code ('ok')
+  METHOD:   Send a GET request with no parameteres.
+  FAIL:     Status code is not 200.
+  """
+  
+  # Act
+  r = requests.get('http://capstonedd.cs.pdx.edu:1337/users')
```

Comment:
Just thinking, but it might be nice if we pull out the host here as a constant. That way if we want to run this against a local build, we can change capstonedd.cs.pdx.edu -> localhost in just one place?